testsuite/test_api.py

```diff
@@ -36,7 +50,10 @@ def wait(self, timeout=900):
             sleep(1)
 
     def destroy(self):
-        return self.client.delete(self.url)
+        return self.client.delete(
+            self.url,
+            HTTP_AUTHORIZATION=f'Bearer {self.token}',
```

Comment:
It should not be necessary to pass `HTTP_AUTHORIZATION` all the time, there must be a way to set default headers on the client object. A quick look at [the source code](https://docs.djangoproject.com/en/2.2/_modules/django/test/client/#Client) suggests there's a `client.defaults` dictionary for that.